ferrocene/ci/configure.sh

```diff
@@ -138,6 +143,39 @@ add --release-description="Ferrocene by Ferrous Systems"
 #                                                                            #
 ##############################################################################
 
+# Set the target used for the build itself (build system, initial compiler
+# stages, etc). This depends on the OS used in CI.
+#
+# If this configuration is missing or changed, the wrong build platform will be
+# used by the build system.
+if [[ "${FERROCENE_HOST:-}" = "x86_64-pc-windows-msvc" ]]; then
+    # In theory, when run under MSYS the default ABI would be GNU, not MSVC.
+    # We're running under MSYS just for compatibility with our CI scripts, not
+    # because we target the GNU ABI, so switching the build platform to the
+    # MSVC one is correct.
+    add --build="x86_64-pc-windows-msvc"
+elif [[ "${FERROCENE_HOST:-}" = "x86_64-unknown-linux-gnu" ]]; then
+    add --build="x86_64-unknown-linux-gnu"
+elif [[ "${FERROCENE_HOST:-}" = "aarch64-apple-darwin" ]]; then
+    add --build="aarch64-apple-darwin" # We only build on ARM Macs right now!
+fi
```

Comment:
This seems redundant with the `FERROCENE_BUILD_HOST` environment variable. Also, if the content of the `FERROCENE_BUILD_HOST` environment variable would be the same as the one bootstrap auto-detects, we should be able to skip setting it.

I'd do something like this here:

```bash
if [[ -x "${FERROCENE_BUILD_HOST+x}" ]]; then
    add "--build=${FERROCENE_BUILD_HOST}"
fi
```